api/app/src/routes/medical/schemas/organization.ts

```diff
@@ -1,13 +1,16 @@
 import { z } from "zod";
 import { OrgType } from "../../../models/medical/organization";
 import { addressSchema } from "./address";
+import { baseUpdateSchema } from "./base-update";
 
 export const orgTypeSchema = z.nativeEnum(OrgType);
 
-export const organizationSchema = z.object({
+export const organizationCreateSchema = z.object({
   name: z.string().min(1),
   type: orgTypeSchema,
   location: addressSchema,
 });
+export type OrganizationCreate = z.infer<typeof organizationCreateSchema>;
 
-export type Organization = z.infer<typeof organizationSchema>;
+export const organizationUpdateSchema = organizationCreateSchema.merge(baseUpdateSchema);
```

Comment:
Update schema extending from baseUpdateSchema (which defined the `eTag` inbound/payload prop).